WebDevStudios/wd_s
# Theme/extras hooks cleanup

### DESCRIPTION ###
Cleans up some functions in and around our `/inc/` directory.

- Removed a placeholder image function that no longer seems to work
- Moved a couple of hooks from `extras.php` into `hooks.php`
- Moved an ACF helper from `extras.php` into `acf.php`
- Sets default width and height on the Unsplash placeholder otherwise, the function breaks if no arguments are passed in
- Removes commented out sidebar registration code
- Whitelists the debug mode check to avoid the PHPCS `nonce verification` warning

### OTHER ###

### STEPS TO VERIFY ###
Nothing should have changed on the frontend; I tested the Unsplash function and checked to make sure header and footer scripts still displayed as expected. They do! So, maybe test those out as well just to verify.

### DOCUMENTATION ###
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

Yes! In an effort to beef up our wiki, I've already started adding and cleaning up pages like Extras and Hooks, so once this is merged in we can add a task to wrap up the rest of the documentation around these files.
